### ISSUE TITLE
Unit tests fails on Python 3.13 for 4.11.0

### ISSUE THREAD

**Issue by hegjon** [python/typing_extensions#370]
```
+ cd typing_extensions-4.11.0
+ cd src
+ /usr/bin/python3 -m unittest discover
.......................................................................................................................................................s..s..s....................................................................F........................................................................................................................................s.s...............................s........
======================================================================
FAIL: test_generic_protocols_special_from_protocol (test_typing_extensions.ProtocolTests.test_generic_protocols_special_from_protocol)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/typing_extensions-4.11.0/src/test_typing_extensions.py", line 3078, in test_generic_protocols_special_from_protocol
    self.assertEqual(typing_extensions._get_protocol_attrs(P), {'meth'})
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: Items in the first set but not the second:
'__static_attributes__'

----------------------------------------------------------------------
Ran 406 tests in 0.192s

FAILED (failures=1, skipped=6)

RPM build errors:
error: Bad exit status from /var/tmp/rpm-tmp.ghdBXH (%check)
    Bad exit status from /var/tmp/rpm-tmp.ghdBXH (%check)
Finish: rpmbuild python-typing-extensions-4.11.0-1.fc41.src.rpm
Finish: build phase for python-typing-extensions-4.11.0-1.fc41.src.rpm
ERROR: Exception(python-typing-extensions-4.11.0-1.fc41.src.rpm) Config(fedora-rawhide-python313) 4 minutes 5 seconds
INFO: Results and/or logs in: /var/lib/mock/fedora-rawhide-python313/result
ERROR: Command failed:
 # /usr/bin/systemd-nspawn -q -M 5f6e039040294ebead1e397a017058a6 -D /var/lib/mock/fedora-rawhide-python313/root -a -u mockbuild --capability=cap_ipc_lock --bind=/tmp/mock-resolv.kntf21v9:/etc/resolv.conf --bind=/dev/btrfs-control --bind=/dev/mapper/control --bind=/dev/fuse --bind=/dev/loop-control --bind=/dev/loop0 --bind=/dev/loop1 --bind=/dev/loop2 --bind=/dev/loop3 --bind=/dev/loop4 --bind=/dev/loop5 --bind=/dev/loop6 --bind=/dev/loop7 --bind=/dev/loop8 --bind=/dev/loop9 --bind=/dev/loop10 --bind=/dev/loop11 --console=pipe --setenv=TERM=vt100 --setenv=SHELL=/bin/bash --setenv=HOME=/builddir --setenv=HOSTNAME=mock --setenv=PATH=/usr/bin:/bin:/usr/sbin:/sbin '--setenv=PROMPT_COMMAND=printf "\033]0;<mock-chroot>\007"' '--setenv=PS1=<mock-chroot> \s-\v\$ ' --setenv=LANG=C.UTF-8 --resolv-conf=off bash --login -c '/usr/bin/rpmbuild -ba --noprep --noclean --target x86_64 /builddir/build/SPECS/python-typing-extensions.spec'
```

**Comment by AlexWaygood** [comment]
Hiya -- we already fixed this on `main` via #367

**Comment by hegjon** [comment]
Version: `Python 3.13.0a6`
